src/components-creators-insights/sq-modal-activation/sq-modal-activation.component.tsx

```
@@ -148,21 +155,62 @@ export default ({
+                  icon={profile?.hasCreatorsInsights ? 'fa-solid fa-info-circle' : 'fa-solid fa-triangle-exclamation'}
+                  color={profile?.hasCreatorsInsights ? 'var(--blue-30)' : 'var(--red-30)'}
+                />
+              </span>
+              <div className="wrapper-selectors toggle">
+                <SqSelector
+                  type="checkbox"
+                  name={`toggle-public-profile-${profile.username}`}
+                  id={`toggle-public-profile-${profile.profileId}`}
+                  checked={profile?.isSharedCreatorsInsights}
+                  onChange={() =>
+                    onTogglePublicProfile && onTogglePublicProfile(profile.profileId, profile.socialNetwork, profile.hasCreatorsInsights)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)